src/components/Dashboard/FilingHistoryList/CompletedAlteration.vue

```diff
@@ -2,10 +2,10 @@
   <div v-if="filing" class="completed-alteration-details body-2">
     <h4>Alteration Complete</h4>
 
-    <p v-if="filing.newLegalType !== filing.oldLegalType">
-      {{entityName || 'This company'}} was successfully altered
-      from a {{filing.oldLegalType}} to a {{filing.newLegalType}}
-      on {{filing.effectiveDateTime}} Pacific Time.
+    <p v-if="filing.toLegalType !== filing.fromLegalType">
```

Comment:
This is nice clarity. Is this a direct representation of property value names from the api?

Reply:
Yes. In Filing History List, we have:
```
item.toLegalType = filing.data.alteration?.toLegalType || null
item.fromLegalType = filing.data.alteration?.fromLegalType || null
```

However, this raises the idea of using the list from the API directly in the template, instead of converting it into `historyItems`. I thought that would be too much churn so I didn't pursue it.

Reply:
The main drivers for this refactor are:
- API provides more ready-to-use data (so UI doesn't need lots of business logic)
- fetching the data is faster since less data is provided up front (eg, the comments and documents are loaded on demand)
- probably lots of API cleanup (and, obviously, UI cleanup too 😀 )